apps/src/gamelab/GameLab.js

```diff
@@ -822,8 +824,13 @@ GameLab.prototype.initInterpreter = function () {
   this.JSInterpreter.onExecutionError.register(this.handleExecutionError.bind(this));
   this.consoleLogger_.attachTo(this.JSInterpreter);
   getStore().dispatch(jsDebugger.attach(this.JSInterpreter));
+  let code = this.studioApp_.getCode();
+  if (this.level.useHelperLib) {
```

Comment:
Should userHelperLib be an array, to support more than one lib in the future?

Reply:
+1

Reply:
Done, but it's a little ugly.

It'd be great if the webpack rule that parses `.interpreted.js` files could automatically generate the `LIBRARIES` map instead of having to hardcode it, but I couldn't find a straightforward way to do that. I'll just punt on making that nicer for now.